mlir/test/Quantum/IonsDecompositionTest.mlir

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// RUN: quantum-opt --ions-decomposition --split-input-file -verify-diagnostics %s | FileCheck %s
+// RUN: quantum-opt --pass-pipeline="builtin.module(ions-decomposition{func-name=test_ions_decomposition})"  --split-input-file -verify-diagnostics %s | FileCheck %s
```

Comment:
Is the pass-pipeline part necessary, i.e. are we not able to include pass options when invoking the pass directly?

Reply:
Let me try, good suggestion